happi/tests/test_cli.py

```diff
@@ -649,9 +649,10 @@ def test_load_glob_args(
 
     with mock.patch.object(IPython, 'start_ipython') as m:
         _ = runner.invoke(
-            happi_cli, ['--path', happi_cfg, 'load', 'tst_*']
+            happi_cli, ['--path', happi_cfg, 'load', 'tst_*', 'device_class=types*']
```

Comment:
Is this addition fixing the previous test or testing a second use-case?  Should we have a separate test case that tests multiple glob arguments?

It never hurts to have more independent tests

Reply:
Hmm I guess this was a second use case.  I'll create a separate test case for multiple glob arguments.